store/src/main/java/com/buschmais/jqassistant/core/store/impl/AbstractGraphStore.java

```diff
@@ -58,7 +58,7 @@ public void start() {
             .classLoader(ofNullable(storePluginRepository.getClassLoader()))
             .types(storePluginRepository.getDescriptorTypes())
             .validationMode(ValidationMode.NONE)
-            .clearAfterCompletion(false)
+     //       .clearAfterCompletion(false)
```

Comment:
?

Reply:
Change required for debugging and accidentially committed . Now reverted.